Editing pass for 0.17 release notes (#20665)
bevyengine/bevy: release-content/release-notes/atmosphere_environment_map.md

---

> title: Generate environment maps from the procedural atmosphere

Reviewer: ```suggestion
title: Generate environment maps from a procedural atmosphere
```

authors: ["@mate-h"]
pull_requests: [20529]
---

(TODO: Embed screenshot of atmosphere-generated reflections)

As your procedural sky changes, the reflections and ambient light in your scene should automatically be updated to match.

> Now, they can be — no pre-baked environment maps needed.

Reviewer: The two sentence split here is a little weird. Maybe talk about how there was no option in Bevy 0.16?

Author: ```suggestion
Now, this just works!
This is fully dynamic: no pre-baked environment maps are needed.
```

---

> title: Generate environment maps from a procedural atmosphere

Reviewer 2: ```suggestion
title: Runtime generated environment maps
```
This is not specific to atmospheres, and the point is that you can now generate them on the fly. This means you can do IBL from in-scene dynamic reflection probes, or user-uploaded environment maps, not just atmosphere!

Im not sure what the best way to communicate this is, but i'd like to highlight this is not atmosphere specific.

Reviewer 2: Actually oops, is there a different release note for that? Maybe we want to unify them.

Author: Yeah, we can merge those 🤔
